supervisor/shared/board_busses.c

```diff
@@ -91,7 +95,7 @@ MP_DEFINE_CONST_FUN_OBJ_0(board_spi_obj, board_spi);
 #if BOARD_UART
 STATIC mp_obj_t uart_singleton = NULL;
 
-STATIC mp_obj_t board_uart(void) {
+mp_obj_t board_uart(void) {
     if (uart_singleton == NULL) {
         busio_uart_obj_t *self = m_new_obj(busio_uart_obj_t);
```

Comment:
Make this long-lived?

Reply:
I'd rather not in this PR. It already has a lot of extra stuff in it.